cmd/head/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+
+	var lines int
+	flag.IntVar(&lines, "n", 10, "custom number of lines to print")
+	flag.Parse()
+
+	var fileName string
+	var input io.Reader
+	var stdLines []string
+
+	if flag.NFlag() == 0 {
```

Comment:
same here for flag.Args

Reply:
> same here for flag.Args

Nflags directly uses flag.Args length I guess